pkg/harvester/edit/harvesterhci.io.networkattachmentdefinition.vue

```diff
@@ -94,13 +94,25 @@ export default {
         };
       });
     },
+
+    networkType() {
+      return ['L2VlanNetwork', 'UntaggedNetwork'];
+    },
+
+    isUntaggedNetwork() {
+      if (this.isView) {
+        return this.value.vlanType === 'UntaggedNetwork';
```

Comment:
The type of detail page is wrong


https://user-images.githubusercontent.com/18737885/196844498-4b1dbd85-d098-4f7f-8562-3de97a05acea.mov